File: spec/code_samples/Python python3/keys@{keyId}@activate/put.py
import http.client

conn = http.client.HTTPConnection("ec2-3-222-208-106.compute-1.amazonaws.com:8086")

payload = "{\"allocation\":\"TIER4\"}"

headers = { 'content-type': "application/json" }

conn.request("PUT", "/keys/%7BkeyId%7D/activate", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: spec/code_samples/Node unirest/exchange-keys@{keyId}@activate/put.js
var unirest = require("unirest");

var req = unirest("PUT", "http://ec2-3-222-208-106.compute-1.amazonaws.com:8086/exchange-keys/%7BkeyId%7D/activate");

req.headers({
  "content-type": "application/json"
});

req.type("json");
req.send({
  "allocation": "TIER4"
});

req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});
